manifests/install.pp

```diff
@@ -6,13 +6,12 @@
 #
 class python::install {
 
-  $python_version = getparam(Class['python'], 'version')
-  $python = $python_version ? {
-    'system'                        => 'python',
-    'pypy'                          => 'pypy',
-    /\A(python)?([0-9](\.?[0-9])+)/ => "python${2}",
-    /\Arh-python[0-9]{2}/           => $python_version,
-    default                         => "python${python::version}",
+  $python = $::python::version ? {
+    'system'              => 'python',
+    'pypy'                => 'pypy',
+    /\A(python)?([0-9]+)/ => "python${2}",
+    /\Arh-python[0-9]{2}/ => $::python::version,
+    default               => "python${::python::version}",
```

Comment:
can you please undo the leading `::`? From our [review guidelines](https://voxpupuli.org/docs/#reviewing-a-module-pr):
> In the majority of cases, variables shouldn’t be accessed via topscope: `$::modulename::$param`. Instead do: `$modulename::$param`

Reply:
Done.